src/modules/App/DesktopLayout.tsx

```diff
@@ -70,6 +76,8 @@ export const DesktopLayout: React.FC<DesktopLayoutProps> = (props: DesktopLayout
     selectedChannelUrl: currentChannel?.url,
     onChannelCreated: updateFocusedChannel,
     onUserProfileUpdated: onProfileEditSuccess,
+    isTypingIndicatorEnabled: isTypingIndicatorEnabledOnChannelList,
+    isMessageReceiptStatusEnabled: isMessageReceiptStatusEnabledOnChannelList,
```

Comment:
Maybe.. the global-level configs shouldn't be injected into the module-level settings

Reply:
This is a DesktopLayout component, and it should accept the global level options, if not it won't work as our expect

So, the current issue is that the two options `isTypingIndicatorEnabledOnChannelList` and `isMessageReceiptStatusEnabledOnChannelList` are not making any impact.

Reply:
It sounds like the module isn't adhering to those global settings. Could you please confirm that part?